dev/fillInQAHistory.sql

```diff
@@ -0,0 +1,193 @@
+(SELECT (new_data::json->>'id')::bigint as certification_result_id,
+CASE WHEN (SELECT count(*) FROM ((SELECT * FROM json_to_recordset(original_data::json->'certificationResults') as results(number text, g1success text)) EXCEPT ALL (SELECT * FROM json_to_recordset(new_data::json->'certificationResults') as results(number text, g1success text))) diff) > 0 
+THEN (SELECT string_agg(number || ':' || g1success, ';') FROM ((SELECT * FROM json_to_recordset(original_data::json->'certificationResults') as results(number text, g1success text)) EXCEPT ALL (SELECT * FROM json_to_recordset(new_data::json->'certificationResults') as results(number text, g1success text))) diff) END as new, creation_date, last_modified_user as activity_user_id, last_modified_user 
+FROM openchpl.activity WHERE activity_object_concept_id = (SELECT activity_concept_id FROM openchpl.activity_concept WHERE concept = 'CERTIFIED_PRODUCT')) as changed_name
+WHERE new IS NOT NULL)
+
+/* TODO: 6 - Measure Successfully Tested for 170.314 (g)(2) Edited */
+INSERT INTO openchpl.questionable_activity_certification_result (questionable_activity_trigger_id, certification_result_id, before_data, after_data, activity_date, activity_user_id, last_modified_user, creation_date, last_modified_date)
+(SELECT (SELECT id FROM openchpl.questionable_activity_trigger WHERE name = 'Measure Successfully Tested for 170.314 (g)(2) Edited'), certification_result_id, null, new, creation_date, activity_user_id, last_modified_user, creation_date, creation_date FROM 
+(SELECT (new_data::json->>'id')::bigint as certification_result_id,
+CASE WHEN (SELECT count(*) FROM ((SELECT * FROM json_to_recordset(original_data::json->'certificationResults') as results(number text, g2Success text)) EXCEPT ALL (SELECT * FROM json_to_recordset(new_data::json->'certificationResults') as results(number text, g2Success text))) diff) > 0 
+THEN (SELECT string_agg(number || ':' || g2Success, ';') FROM ((SELECT * FROM json_to_recordset(original_data::json->'certificationResults') as results(number text, g2Success text)) EXCEPT ALL (SELECT * FROM json_to_recordset(new_data::json->'certificationResults') as results(number text, g2Success text))) diff) END as new, creation_date, last_modified_user as activity_user_id, last_modified_user 
+FROM openchpl.activity WHERE activity_object_concept_id = (SELECT activity_concept_id FROM openchpl.activity_concept WHERE concept = 'CERTIFIED_PRODUCT')) as changed_name
+WHERE new IS NOT NULL)
+
+/* TODO: 7 - Measure Successfully Tested for 170.315 (g)(1) Added */
+INSERT INTO openchpl.questionable_activity_certification_result ((SELECT id FROM openchpl.questionable_activity_trigger WHERE name = 'Measure Successfully Tested for 170.315 (g)(1) Added'), certification_result_id, null, "170.315 (b)(3):EC Individual (TIN/NPI)", 2017-09-10 18:52:30.569, 5, 5, 2017-09-10 18:52:30.569, 2017-09-10 18:52:30.569)
```

Comment:
Does this work? I feel like the syntax is not right. Where is certification_result_id defined or selected? Do the dates need to be in quotes? Didn't we find more than just 1 instance of g1 macra measures added on prod/stg?